src/itemTypes/common.ts

```diff
@@ -214,7 +220,7 @@ export function updateItemData (
       updateResp => {
         resolve(id);
       },
-      () => reject()
+      () => reject({ success: false })
```

Comment:
Maybe I'm missing it from just seeing this little bit...why is this only rejecting with `{ success: false }`

Reply:
Just hadn't gotten to updating the all of the rejects to the new form